### PR TITLE
[CFM-1414-fix] Fix the wellcome to group email notification

### PR DESCRIPTION
**Pull request by anvmn**
New function created = > used both for  open and moderated groups wellcome notification.

https://issuetracker.amplexor.com/jira/browse/CFM-1414

### CODE REVIEW

**Comment by anvmn** [comment]
@ordavidil 
Ready for review

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -21,14 +21,8 @@ function c4m_user_notifications_moderated_group_new_membership_request($entity)
  * Email notification to user about acceptance to moderated group.
  */
 function c4m_user_notifications_moderated_group_notify_user_membership_accepted($membership) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: Remove this function.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 2]
```diff
@@ -41,6 +35,8 @@ function c4m_user_notifications_moderated_group_notify_admins_membership_accepte
  * Email notification to user about acceptance to open group.
  */
 function c4m_user_notifications_open_group_notify_user_membership_accepted($membership) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: Remove this function.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 3]
```diff
@@ -241,3 +237,19 @@ function c4m_user_notifications_notify_inactive_users() {
  */
 function c4m_user_notifications_notify_comment_on_content($comment) {
 }
+
+/**
+ * Helper function.
+ *
+ * Email notification to user about acceptance to  group (moderated and open).
+ */
+function _c4m_user_notifications_notify_user_membership_accepted($membership) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: Remove the `_` prefix, and you this function on the hook itself.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 4]
```diff
@@ -241,3 +237,19 @@ function c4m_user_notifications_notify_inactive_users() {
  */
 function c4m_user_notifications_notify_comment_on_content($comment) {
 }
+
+/**
+ * Helper function.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: Remove this line.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 5]
```diff
@@ -241,3 +237,19 @@ function c4m_user_notifications_notify_inactive_users() {
...
+ *
+ * Email notification to user about acceptance to  group (moderated and open).
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: Fix the docs.

**Pull request merged**